app/frontend/components/pages/txHistory/transactionHistory.tsx

```diff
@@ -114,13 +120,22 @@ const ExportCSV = ({transactionHistory, stakingHistory}) => {
     .map((header) => `${header}${delimiter}`)
     .join('')
 
-  const transactionTypes = {
-    received: 'Received',
-    sent: 'Sent',
-    rewardAwarded: 'Reward awarded',
+  const enum TransactionType {
+    received = 'Received',
+    sent = 'Sent',
+    rewardAwarded = 'Reward awarded',
+  }
+
+  type Entry = {
```

Comment:
I would rename this at least to `TxEntry`

Reply:
Refactored